### PR TITLE
Add command getgroup

### LINKED ISSUE #11: Add new command: [Get Group]
Command: [prefix] getgroup [group]
Description: Show all members of a role/group who are in the discord. ([pref] getgroup board, lists all boardmembers)
Use cases: Let the chair check who are in the discord for meetings, or let users who seek help search for who to contact.
Permissions: Member+ only

### Get group

How the command is invoked `[prefix] [command] [args]`?

[prefix] getgroup [group_name]

### Description

What does the command do and what should `[prefix] help [command]` return to the user?

Show all members of a role/group who are in the discord server. 
([pref] getgroup board, lists all boardmembers their discord tags)

### Use cases

How do you expect the command will be used?

Anyone can see all users who are assigned to the role of a certain group.
Let the chair check who are in the discord for meetings, or let users who seek help search for who to contact.

### Permissions

Who should have access to the command?

[Member+]
Anyone who is a verified Thalia Member should get access to this command.

### Additional context
What does the board think about this command regarding privacy?
Will we send discord names, or direct links to pm if possible?

### PR DESCRIPTION
**Pull request by mark-boute**
Closes #11 

### Summary
Added the command getgroup described in issue #11 

### How to test
Steps to test the changes you made:
1. Run the bot
2. Test the command and it's behavior in the server and in PMs

### CODE REVIEW

File: bot/cogs/groups.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            if ctx.guild:
+                roles = ctx.guild.roles
+                for arg in args:
+                    if arg.lower() in map(lambda r: r.name.lower(), roles):
+                        # get members and pass info
+                        # pylint: disable=cell-var-from-loop
+                        role = find(lambda r: r.name.lower() == arg.lower(), roles)
+                        # pylint: enable=cell-var-from-loop
+
+                        msg = Embed(title=role.name, colour=0xE62272)
```
Comment (mark-boute) [inline bot/cogs/groups.py:37]: Maybe we should have a python file in common for constants such as the colours found in Thalia's style guide

Comment (pingiun) [reply bot/cogs/groups.py:37]: Yes

File: bot/cogs/groups.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+logger = get_logger(__name__)
+
+
+class GroupCog(commands.Cog, name="Group management"):
+    def __init__(self, bot):
+        self.bot = bot
+        logger.info("Group cog initialised")
+
+    @commands.group(
```
Comment (se-bastiaan) [inline bot/cogs/groups.py:16]: Does this really need to be part of a command group? It is currently `!group getgroup <group> <group>` right?

Comment (mark-boute) [reply bot/cogs/groups.py:16]: I wasn't sure where to put it but I felt like a general command group might be useful for commands we might add later (such as a command to move everyone who is in a group to a certain voice channel to start a meeting).
Currently it's `!group getgroup <group(s)>` where `<group(s)>` can be any number of groups separated by a space.

Comment (se-bastiaan) [reply bot/cogs/groups.py:16]: okay that is useful indeed, but semantically it is a double command. So maybe go for `!group info <groups>`?

Comment (mark-boute) [reply bot/cogs/groups.py:16]: Yeah I guess, maybe `!group getmembers <group(s)>`

**Review by se-bastiaan — APPROVED** [review]

**Pull request merged**